src/pages/pages.scss

```
@@ -89,6 +89,46 @@
 
 }
 
+.mapAddins {
+
+  &__docs-service{
+    margin: 1rem 0;
+    min-width: 0;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-clip: border-box;
+    word-wrap: break-word;
+    border-radius: 0.25rem;
+    border: 1px solid rgba(0,0,0,.125);
+  }
+
+  &__docs-service__part {
+    flex: 1 1 auto;
+    padding: 1.25rem;
+    border-top: 1px solid rgba(0,0,0,.125);
+  }
+
+  &__docs-service__part:first-child {
+    border-top: none;
+  }
+
+  &__table-container{
+    display: inline-block;
+    overflow: hidden;
+    border-radius: 4px;
+    border: 1px solid $table-border;
+  }
+
+  &__table {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: auto;
+    margin: 0;
+  }
```

```suggestion
  &__table-container{
    @extend .table-container;
    display: inline-block !important;
    overflow: hidden !important;
  }

  &__table {
    width: auto;
    margin: 0;
  }
```

These seem to be redundant. We already have styles defined for .table-container & table. We should reuse those as a base and then use these ones for the specific differences for this page. For mapAddins__table-container, is the display & overflow used for anything as they're not really making a huge difference if enabled? So maybe we just use the .table-container class instead (unless there's a specific reason you see the reason of having it separate)?